posts/20230622/index.qmd

```diff
@@ -0,0 +1,36 @@
+---
+title: "Let’s Verify Step by Step"
+author:
+    - Akshata Mohan
+    - Hongsup Shin
+date: "2023-06-22"
+image: "fig1.png"
+description: "TBD"
+categories: [paper]
+---
+
+:::{.callout-note}
+- Paper: [*Let’s Verify Step by Step*](https://arxiv.org/abs/2305.20050)
+- Presenter: [Akshata](https://www.linkedin.com/in/akshata-mohan/)
+- Attendees: [Hongsup](https://www.linkedin.com/in/hongsupshin/), [Meghann](https://www.linkedin.com/in/meghann-agarwal/), [Saina](https://www.linkedin.com/in/saina199/)
+:::
+
+## Why this paper?
```

Comment:
"This is an interesting new paper by OpenAI that discusses how we can apply the same principles we use to solve math problems to AI. It's interesting to see how a process based supervision approach is superior to outcome based supervision approach"